.github/workflows/check.yml

```diff
@@ -22,14 +22,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
-        name: Configuring nodejs 16.x version
+        name: Configuring nodejs 18.x version
```

Comment:
I noticed we’re moving to `18.x` here, but the latest available release is Node.js `24.x` https://github.com/actions/setup-node#supported-version-syntax. Any specific reason why we're upgrading to this version?